### PR TITLE
Updated docs with enterprise update details

### PR DESCRIPTION
**Pull request by Tanuja-Kulal**

### CODE REVIEW

**Comment by PRMerger18** [comment]
@Tanuja-Kulal : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/install/set-defaults-for-enterprise-deployments.md [hunk 1]
```diff
@@ -50,6 +50,10 @@ You can set the following registry values:
 | `KeepDownloadedPayloads` | `REG_DWORD` | 1 | Keep package payloads even after they are installed. You can change the value anytime. Disabling the policy removes any cached package payloads for the instance you repair or modify. For more information, see the [Disable or move the package cache](disable-or-move-the-package-cache.md) page. |
 | `SharedInstallationPath` | `REG_SZ` or `REG_EXPAND_SZ` | %ProgramFiles(x86)%\Microsoft Visual Studio\Shared | The directory where some packages shared across versions of instances of Visual Studio are installed. You can change the value any time, but it will only affect future installs. Any products already installed to the old location must not be moved or they might not function correctly. |
 | `BackgroundDownloadDisabled` |`REG_DWORD` | 1 | Prevent setup from downloading updates automatically for all installed Visual Studio products. You can change the value anytime. |
+| `AdministratorUpdatesEnabled` | `REG_DWORD` | 1 | If missing or is set to 0, Administrator Updates will be blocked from applying to the client computer. This is for administrative use. For more information, see the [Administrative Updates](http://TODOLink) page. |
```
Comment (JoshuaPartlow) [inline docs/install/set-defaults-for-enterprise-deployments.md:53]:
```suggestion
| `AdministratorUpdatesEnabled` | `REG_DWORD` | 1 | Allows Administrator Updates to be applied to the client computer. If this value is missing or is set to 0, Administrator Updates will be blocked. This value is for administrative use. For more information, see the [Administrative Updates](http://TODOLink) page. | 
```
```suggestion
| `AdministratorUpdatesEnabled` | `REG_DWORD` | 1 | If missing or is set to 0, Administrator Updates will be blocked from applying to the client computer. This is for administrative use. For more information, see the [Administrative Updates](http://TODOLink) page. | 
```

File: docs/install/set-defaults-for-enterprise-deployments.md [hunk 2]
```diff
@@ -50,6 +50,10 @@ You can set the following registry values:
...
 | `SharedInstallationPath` | `REG_SZ` or `REG_EXPAND_SZ` | %ProgramFiles(x86)%\Microsoft Visual Studio\Shared | The directory where some packages shared across versions of instances of Visual Studio are installed. You can change the value any time, but it will only affect future installs. Any products already installed to the old location must not be moved or they might not function correctly. |
 | `BackgroundDownloadDisabled` |`REG_DWORD` | 1 | Prevent setup from downloading updates automatically for all installed Visual Studio products. You can change the value anytime. |
+| `AdministratorUpdatesEnabled` | `REG_DWORD` | 1 | If missing or is set to 0, Administrator Updates will be blocked from applying to the client computer. This is for administrative use. For more information, see the [Administrative Updates](http://TODOLink) page. | 
+| `AdministratorUpdatesOptOut` | `REG_DWORD` | 1 | The absence of the registry value, or a set value of 0, means that the Visual Studio user wants to receive administrator updates to Visual Studio. This is for developer user (if they have permissions). For more information, see the [Administrative Updates](http://TODOLink) page. |
```
Comment (JoshuaPartlow) [inline docs/install/set-defaults-for-enterprise-deployments.md:54]:
```suggestion
| `AdministratorUpdatesOptOut` | `REG_DWORD` | 1 | Indicates that the user does not want to receive administrator updates to Visual Studio. The absence of the registry value, or a set value of 0, means that the Visual Studio user wants to receive administrator updates to Visual Studio. This is for developer user (if they have permissions). For more information, see the [Administrative Updates](http://TODOLink) page. | 
```
```suggestion
| `AdministratorUpdatesOptOut` | `REG_DWORD` | 1 | The absence of the registry value, or a set value of 0, means that the Visual Studio user wants to receive administrator updates to Visual Studio. This is for developer user (if they have permissions). For more information, see the [Administrative Updates](http://TODOLink) page. | 
```

File: docs/install/update-a-network-installation-of-visual-studio.md [hunk 3]
```diff
@@ -102,6 +102,8 @@ Depending on how your network environment is configured, an update can either be
 
 ::: moniker-end
 
+* Administrators in organizations which use enterprise tools like Windows Server Update Services and System Center Configuration Manager can update client deployments of Visual Studio with [Updates using Microsoft Update Catalog](http://TODOLink). These updates are designed to work with client deployments of Visual Studio which were created from a network layout.
```
Comment (JoshuaPartlow) [inline docs/install/update-a-network-installation-of-visual-studio.md:105]:
```suggestion
* Administrators in organizations that use enterprise tools such as Windows Server Update Services and System Center Configuration Manager can update client deployments of Visual Studio with [Updates using Microsoft Update Catalog](http://TODOLink). These updates are designed to work with client deployments of Visual Studio that were created from a network layout. 
```

File: docs/install/update-visual-studio.md [hunk 4]
```diff
@@ -281,6 +281,10 @@ Here's how:
 
 ::: moniker-end
 
+## Administrative Updates 
+
+If you are part of an organization with centralized enterprise management of your computer, your enterprise administrator may update your machine for you. [Updates using Microsoft Update Catalog](http://TODOLink) can be controlled with configuration settings allowing your computer to accept or refuse these types of updates.
```
Comment (JoshuaPartlow) [inline docs/install/update-visual-studio.md:286]:
```suggestion
If you are part of an organization with centralized enterprise management of your computer, your enterprise administrator can update your machine for you. [Updates using Microsoft Update Catalog](http://TODOLink) can be controlled with configuration settings allowing your computer to accept or refuse these types of updates.  
```

Comment (JoshuaPartlow) [reply docs/install/update-visual-studio.md:286]: Depending on what you're trying to convey, I'd suggest either _can_ or _might_.

File: docs/install/visual-studio-administrator-guide.md [hunk 5]
```diff
@@ -52,6 +52,8 @@ Before you deploy Visual Studio across your organization, there are a few decisi
 
 * Decide if you want to enable Group Policy, and if you want to configure Visual Studio to disable customer feedback on individual computers.
 
+* For organizations which utilize enterprise deployment tools and utilize the Microsoft Update Catalog and/or Windows Server Update Services, decide if you will take advantage of Visual Studio Security Updates, Quality Updates and/or Feature Updates available to such tools. For more information see [Updates using Microsoft Update Catalog](http://TODOLink).
```
Comment (JoshuaPartlow) [inline docs/install/visual-studio-administrator-guide.md:55]:
```suggestion
* For organizations that utilize enterprise deployment tools, as well as the Microsoft Update Catalog and/or Windows Server Update Services, decide if you will take advantage of Visual Studio Security Updates, Quality Updates, and/or Feature Updates available to such tools. For more information see [Updates using Microsoft Update Catalog](http://TODOLink). 
```

File: docs/install/visual-studio-enterprise-guide.md [hunk 6]
```diff
@@ -59,8 +59,11 @@ Learn how to update Visual Studio successfully and fix update issues.
 
 - **Follow [Windows security baselines](/windows/security/threat-protection/windows-security-baselines)**. Microsoft is dedicated to providing its customers with secure operating systems, such as Windows 10 and Windows Server, and secure apps, such as Microsoft Edge. In addition to the security assurance of its products, Microsoft also enables you to have fine control over your environments by providing various configuration capabilities. 
 
+- **[Update Visual Studio using enterprise deployment tools like the Microsoft Update Catalog, Windows Server Update Services & SCCM](http://TODOLink). For IT Admins who want to manage deployments across their organization, Security Updates, Quality Updates and Feature Updates are available.
```
Comment (JoshuaPartlow) [inline docs/install/visual-studio-enterprise-guide.md:62]:
```suggestion
- **[Update Visual Studio using enterprise deployment tools like the Microsoft Update Catalog, Windows Server Update Services & SCCM](http://TODOLink). For IT Admins that want to manage deployments across their organization, Security Updates, Quality Updates and Feature Updates are available. 
```

**Comment by PRMerger15** [comment]
@Tanuja-Kulal : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger15** [comment]
@Tanuja-Kulal : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger16** [comment]
@Tanuja-Kulal : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger16** [comment]
@Tanuja-Kulal : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger16** [comment]
@Tanuja-Kulal : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger16** [comment]
@Tanuja-Kulal : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger16** [comment]
@Tanuja-Kulal : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger16** [comment]
@Tanuja-Kulal : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger16** [comment]
@Tanuja-Kulal : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ornellaalt** [comment]
It's unclear to me what the TODO link would be here. The two new pages have now been merged and published, so you can replace those with the active links.

**Comment by PRMerger15** [comment]
@Tanuja-Kulal : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger13** [comment]
@Tanuja-Kulal : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger9** [comment]
@Tanuja-Kulal : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger9** [comment]
@Tanuja-Kulal : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger10** [comment]
@Tanuja-Kulal : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by v-andreaco** [comment]
@Tanuja-Kulal @JoshuaPartlow @ornellaalt 

The changes to these articles looks extensive. Will you please move this to the private repo (visualstudios-docs-pr) so that Acrolinx and OPS validation can run and staging can be verified?

Thanks!

**Comment by Tanuja-Kulal** [comment]
Moved to private repo

**Pull request closed without merging**